spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "rails_helper"
```

Comment:
Tiny linting nitpick - we probably want a new line here before the describe block